extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -132,10 +130,7 @@ class AmpStickyAd extends AMP.BaseElement {
       this.removeOnScrollListener_();
       this.deferMutate(() => {
         this.visible_ = true;
-        this.element.classList.add('-amp-sticky-ad-visible');
         this.viewport_.addToFixedLayer(this.element);
-        this.updateInViewport(dev().assertElement(this.ad_), true);
```

Comment:
These 2 lines have been moved to`#layoutAd_`.  It seems I failed to remove them here and caused them to be called twice. 
https://github.com/ampproject/amphtml/blob/master/extensions/amp-sticky-ad/0.1/amp-sticky-ad.js#L174